CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.32.1](https://github.com/brave/brave-ios/releases/tag/v1.32.1)
+
+- Added CarPlay support for playlist. ([#3673](https://github.com/brave/brave-ios/3673))
```

Comment:
nit: should capitalize `playlist` -> `Playlist` as we're referring to the feature itself in this instance.